pkg/simpleschema/transform.go

```diff
@@ -205,15 +208,52 @@ func (tf *transformer) applyMarkers(schema *extv1.JSONSchemaProps, markers []*Ma
 		case MarkerTypeDescription:
 			schema.Description = marker.Value
 		case MarkerTypeMinimum:
-			if val, err := strconv.ParseFloat(marker.Value, 64); err == nil {
+			if val, err := strconv.ParseFloat(marker.Value, 64); err != nil {
+				return fmt.Errorf("failed to parse minimum enum value: %w", err)
+			} else {
 				schema.Minimum = &val
 			}
 		case MarkerTypeMaximum:
-			if val, err := strconv.ParseFloat(marker.Value, 64); err == nil {
+			if val, err := strconv.ParseFloat(marker.Value, 64); err != nil {
+				return fmt.Errorf("failed to parse maximum enum value: %w", err)
+			} else {
 				schema.Maximum = &val
 			}
+		case MarkerTypeEnum:
+			enumValues := strings.Split(marker.Value, ",")
+
+			var enumJSONValues []extv1.JSON
+
+			for _, val := range enumValues {
+				val = strings.TrimSpace(val)
+				var rawValue []byte
+
+				switch schema.Type {
```

Comment:
Shall we just keep `string` and `integer` here? for the default we can just error saying that enums are only supported for string and integer types